### PR TITLE
Use `noAddress` to limit use of external IPs [VS-1323]

### PR DESCRIPTION
**Pull request by mcovarr**
Use Cromwell's [noAddress](https://support.terra.bio/hc/en-us/community/posts/360060020871-noAddress-true-results-in-stalling-jobs) feature to avoid the unnecessary use of external IPs on Cromwell worker VMs on wide scatters that cause us Google quota problems.

For most of the GVS code this only involves adding `noAddress: true` to the existing runtime attributes. In the PGEN code this was slightly more work to change away from `docker: "ubuntu:22.04"` which is implicitly pulled from Docker Hub. Since `noAddress: true` means the VM can only interact with Google services, we have to switch to a GCR-hosted image as Docker Hub has become unreachable.

- [Mostly successful integration test](https://job-manager.dsde-prod.broadinstitute.org/jobs/32b9e2b5-3c56-4bf8-ab5e-66fa72c7cadb), delta some existing issues with cost discrepancies documented in VS-1324.
- [Successful PGEN extract](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/1f19bea2-a9b9-4ec5-b741-e9f64bbfa35a)

### CODE REVIEW

File: scripts/variantstore/wdl/MergePgenHierarchical.wdl [hunk 1]
```diff
@@ -145,6 +149,8 @@ task MergePgen {
         disks: "local-disk ${disk_in_gb} HDD"
         bootDiskSizeGb: 15
         cpu: "${cpu}"
+        # The PGEN Docker needs to be in GAR or GCR to be able to use noAddress
+        # noAddress: true
```
Comment (rsasch) [inline scripts/variantstore/wdl/MergePgenHierarchical.wdl:153]: Why is this here if it's commented out?

**Review by gbggrant — APPROVED** [review]
Thumbs up, conditional on Bec's comment

File: scripts/variantstore/wdl/MergePgenHierarchical.wdl (hunk 1 above)
Comment (mcovarr) [reply scripts/variantstore/wdl/MergePgenHierarchical.wdl:153]: I was intending to leave behind a breadcrumb so that when VS-1315 (make a PLINK Docker and put it in GCR) is implemented, the author would go back and turn on `noAddress` here. But now that you mention it the VS-1315 is probably a better place for that breadcrumb.

**Review by rsasch — APPROVED** [review]

**Comment by mcovarr** [comment]
@koncheto-broad I added the breadcrumb to the existing VS-1315 ticket to go back and add `noAddress: true` to the `MergePgen` task.
